manager/integration/tests/test_csi.py

```diff
@@ -536,3 +536,119 @@ def test_csi_expansion_with_replica_failure(client, core_api, storage_class, pvc
     write_pod_volume_data(core_api, pod_name, test_data)
     resp = read_volume_data(core_api, pod_name)
     assert resp == test_data
+
+@pytest.mark.skip(reason="TODO") # NOQA
+def test_csi_volumesnapshot_basic():  # NOQA
+    """
+    Test creation / restoration / deletion of a backup via the csi snapshotter
+
+    Context:
+
+    We want to allow the user to programmatically create/restore/delete longhorn backups
+    [csi snapshot mechanism](https://kubernetes.io/docs/concepts/storage/volume-snapshots/)
+
+    Setup:
+
+    1. Make sure your cluster contains the below crds
+    https://github.com/kubernetes-csi/external-snapshotter/tree/master/client/config/crd
```

Comment:
Seems we need to add those to our test environment setup.

Reply:
@meldafrawi @khushboo-rancher would you be able to add the necessary components to our test clusters?

Reply:
Added https://github.com/longhorn/longhorn/issues/1787 to track this.